packages/client/hooks/useManualClientSideTrack.ts

```diff
@@ -0,0 +1,23 @@
+import {useCallback} from 'react'
+import SendClientSideEvent from '../utils/SendClientSideEvent'
+import useAtmosphere from './useAtmosphere'
+
+let eventId = 0
+
+const useManualClientSideTrack = () => {
+  const atmosphere = useAtmosphere()
+
+  const trackEvent = useCallback(
+    (event: string, options: Record<string, any>) => {
+      SendClientSideEvent(atmosphere, event, {
+        ...options,
+        eventId: ++eventId
```

Comment:
Why do we need `eventId` here?

Reply:
We have it in the other hook as there seemed to be a bug where some users were sending it non-stop, and we were trying to debug whether it was happening from that hook or elsewhere. 

If it is happening in the other hook, it's more likely to be related to the `useEffect` which isn't in this hook. I'll remove it, and if the event fires non-stop too, I'll add eventId back.